lib/glimesh_web/live/user_live/stream.ex

```diff
@@ -104,13 +104,18 @@ defmodule GlimeshWeb.UserLive.Stream do
   end
 
   def handle_event("lost_packets", %{"uplink" => uplink, "lostPackets" => lostPackets}, socket) do
+    message =
+      if lostPackets > 3,
+        do:
+          gettext(
+            "We're detecting some networking problems between you and the streamer. You may experience video drops, jitter, or other issues!"
```

Comment:
Possibly add a "This is not a bug." to the end of the message to cut down on people just posting the message in the bugs channel. Either way, good to go from me :) 

Dan might have some more input since he's got the big brain.

Reply:
No no no it's clone with the big 🧠 ideas.

Reply:
I'd suggest raising the threshold though to at least 10, or honestly even higher.

When I've seen this message it's bursts of 5+ packets and I have a decent network.

And eventually it'd be nice to measure lost over expected percent, packet inter arrival jitter, etc

Reply:
Alternatively go with the super subtle route of changing the "signal" icon to one indicating no signal. Given this notification isn't really actionable to the viewer or streamer I don't think we need to be in their face with it.

Reply:
Updated the threshold to 6 instead, I do want to keep some user visible message around though until we introduce NACKs. The Chrome browser especially does not handle these dropped packets at all, and will unload the video client making for a confusing experience for the viewer.